README.md

```diff
@@ -62,6 +62,18 @@ into their own mount. This could be achieved with a configuration like:
                 - %{environment}
                 - secret
 
+## Default field
```

Comment:
This section probably makes more sense to live under the "Lookups" section of the README.
Maybe something like:

```
## Lookups
### Hash
By default...
### Single Value
<your new section about default_field>
```